packages/dd-trace/src/appsec/sdk/index.js

```diff
@@ -1,10 +1,17 @@
 'use strict'
 
 const { trackUserLoginSuccessEvent, trackUserLoginFailureEvent, trackCustomEvent } = require('./track_event')
+const { isUserBlocked } = require('./user_blocking')
+const { block, loadTemplates } = require('../blocking')
+const { getRootSpan } = require('./utils')
+const { storage } = require('../../../../datadog-core')
 
-class AppsecSdk {
-  constructor (tracer) {
+class AppsecSDK {
+  constructor (tracer, config) {
     this._tracer = tracer
+    if (config) {
+      loadTemplates(config)
```

Comment:
Aren't we loading templates twice? In this constructor and when appsec is enabled.

Reply:
I really don't understand why we don't just load the files in config.js like it used to be ??

Reply:
The main reason why I changed it was because whole configuration is sent to telemetry when application starts. We don't want to send the content of the files to backend, we only want to send the configuration.

Reply:
Ok, I will prevent that the templates are loaded more than once and, in a future PR, figure out a better soulution.